README.rst

```diff
@@ -1,23 +1,13 @@
-==================================================
-pyirf |travis| |codacy| |coverage| |documentation|
-==================================================
+=========================================
+pyirf |travis| |coverage| |documentation|
+=========================================
 
```

Comment:
It did not work anymore, it did not appear, I just removed it because it was broken

Reply:
We should change the documentation one to github pages as soon as we merged and the docs are pushed

Reply:
> It did not work anymore, it did not appear, I just removed it because it was broken

on the master's readme it seems to work (https://app.codacy.com/gh/cta-observatory/pyirf/dashboard)
also in this PR is running
I would keep it and see what happens (non a real problem if we fix it later...)

> We should change the documentation one to github pages as soon as we merged and the docs are pushed

Yes sorry I didn't think that we can just change the link

Reply:
also I removed readthedocs, it should not pop out in the next PRs

Reply:
> We should change the documentation one to github pages as soon as we merged and the docs are pushed

Actually we don't need to: the docs are built by the same CI pipeline, so if they fail the travis badge would already take it into account, no?

I would revert the change to have Codacy one (why did you think it didn't work?) and not have one for the docs

By the way, unless I am mistaken, we do not currently have an empty `gh-pages` branch were to deploy the docs when we merge - I think we should do that before merging